app/src/main/java/com/khinthirisoe/bakingapp/ui/baking/view/BakingActivity.kt

```diff
@@ -0,0 +1,77 @@
+package com.khinthirisoe.bakingapp.ui.baking.view
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.ProgressBar
+import android.widget.Toast
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.khinthirisoe.bakingapp.R
+import com.khinthirisoe.bakingapp.data.model.Recipe
+import com.khinthirisoe.bakingapp.di.component.AppComponent
+import com.khinthirisoe.bakingapp.di.component.DaggerActivityComponent
+import com.khinthirisoe.bakingapp.di.module.ActivityModule
+import com.khinthirisoe.bakingapp.ui.baking.BakingContract
+import com.khinthirisoe.bakingapp.ui.base.BaseActivity
+import com.khinthirisoe.bakingapp.ui.ingredients.IngredientsActivity
+import javax.inject.Inject
+
+
+class BakingActivity : BaseActivity(), BakingContract.View, BakingAdapter.BakingRecyclerViewClickListener {
+
+    @Inject
+    lateinit var presenter: BakingContract.Presenter
+
+    private lateinit var bakingRecyclerView: RecyclerView
+    private lateinit var progressBar: ProgressBar
+    private var bakingAdapter: BakingAdapter? = null
+
+    override fun setupComponent(appComponent: AppComponent) {
+        val component =
+            DaggerActivityComponent.builder().appComponent(appComponent).activityModule(ActivityModule(this)).build()
```

Comment:
@KhinThiriSoe better to write it like this. It's more readable
`val component = DaggerActivityComponent.builder()
                                     .appComponent(appComponent)
                                     .activityModule(ActivityModule(this))
                                     .build()`

Reply:
great, it's now combined into one line. :/ 'll edit the comment